docs/user/installation.rst

```diff
@@ -79,24 +76,30 @@ If you want to help developing, run:
     pip install -r requirements.txt
     # install pytorch version for your system (see below)
     pip install -r requirements-dev.txt
-    python setup.py develop
+    pip install -e .
 
     py.test  # unit tests
     pylint skorch  # static code checks
 
 pytorch
 ~~~~~~~
 
-PyTorch is not covered by the dependencies, since the PyTorch
-version you need is dependent on your system. For installation
-instructions for PyTorch, visit the `pytorch website
-<http://pytorch.org/>`__.
+PyTorch is not covered by the dependencies, since the PyTorch version
```

Comment:
That's not true anymore as the `environment.yml` includes pytorch now?

Reply:
I think it's valid this way. Most people will install skorch through pip and `pip install skorch` will not automatically install pytorch. In the section `From source`, it only says to install pytorch in the subsection `Using pip`, not in `Using conda`. For these reasons, I would leave this part as is.